### DISCUSSION TITLE
How to update OpenSSH/LibreSSL? Why not via Windows update?

### DISCUSSION THREAD

**Discussion by m-fessler** [PowerShell/Win32-OpenSSH#2194] [Q&A]
Hello everyone,

a "ssh -V" returns the following on a fully patched Windows 10 22H2:

> OpenSSH_for_Windows_8.1p1, LibreSSL 3.0.2

What is the easiest way to update OpenSSH and LibreSSL and keep them up to date in the future?
Why isn't this done via Windows Update?

Thanks and greetings,
Martin

**Comment by maertendMSFT** [comment]
Windows features can only update on a set cadence.  We ship Win32-OpenSSH updates on this repo much more frequently than we are able to update into Windows. I cannot provide details of the Windows update schedule publicly.

Additionally, with our current shipping process, only the latest version of Windows Client and Windows Server receive the latest Win32-OpenSSH versions. Given you are using Windows 10, under our normal process you would not receive another update to Win32-OpenSSH.
We are evaluating other shipping mechanisms that we can support, but I have nothing I can share at this time.
